src/auth/negotiate/kerberos/negotiate_external_acl

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+#
+# This external acl helper checks the Kerberos group information without
+# continues ldap queries by leveraging the kerbors ticket group information
+# i.e. PAC field
+#
+# It requires only during startup aldap query to get the SID for the group name.
```

Comment:
I would rephrase for clarity. For example:

```
During startup, it sends an ldapsearch query to get the SID for the group name.
```

Reply:
That is almost a conjunctive sentence; "During startup, send an LDAP ..." or "Sends an LDAP query during startup to get ..."